fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/JDBCParameterBuildingVisitor.java

```diff
@@ -70,8 +73,10 @@
     // DB2: https://www.ibm.com/support/knowledgecenter/en/SSEPGG_11.5.0/com.ibm.db2.luw.sql.ref.doc/doc/r0001029.html
     // Derby: https://db.apache.org/derby/docs/10.0/manuals/reference/sqlj271.html
     private static final Timestamp SMALLEST_TIMESTAMP = Timestamp.valueOf("0001-01-01 00:00:00.000000");
-    // 23:59:59.999999 used instead of 24:00:00.000000 to ensure it could be represented in FHIR if needed
-    private static final Timestamp LARGEST_TIMESTAMP = Timestamp.valueOf("9999-12-31 23:59:59.999999");
+
+    // 00:00:00.000000 used instead of 24:00:00.000000 to ensure it could be represented in FHIR if needed
+    // Have to account for Timezone shifts
```

Comment:
do we also need to account for timezone shifts in the SMALLEST_TIMESTAMP?0001-01-01 00:00:00.000000

Reply:
very subtle... if you have a timezone in NOT UTC, and shift it -4:00 hours you hit an invalid timezone which is max+4hrs

Reply:
right, but in some places the timezone is +4:00 ...so you'd have the same issue in SMALLEST_TIMESTAMP.
is there no way to tell it to use `"0001-01-01 00:00:00.000000"` and `"9999-12-31 00:00:00.000000"` in UTC?

Reply:
let me think about this one, it might not be as easy as the above...

Reply:
resolved in the latest commits